R/utils.R

```diff
@@ -15,7 +15,30 @@ os_is_windows <- function() {
   if (!is.null(x)) x else y
 }
 
+#' Returns the type of make command to use to compile
+#' @return Returns "mingw32-make" if using cmdstan 2.21+ and running Windows
+#' @noRd
+make_cmd <- function() {
+  # Cmdstan 2.21 introduced TBB that requires mingw32-make on Windows
+  if ((.cmdstanr$VERSION >= "2.21") && os_is_windows) {
```

Comment:
Here and anywhere `os_is_windows` is used it should be `os_is_windows()` because it’s a function that needs to get called

Reply:
Ah, what a goof on my part :) Thanks! Fixed now.